tests/end2end/features/tabs.feature

```diff
@@ -1197,7 +1197,7 @@ Feature: Tab management
         And the following tabs should be open:
             - data/numbers/1.txt (active) (pinned)
 
-    Scenario: :tab-pin open url
+    Scenario: :home on a pinned tab
```

Comment:
Just a heads up, I'm overloading this PR a bit to fix a small typo in a test I added a bit earlier.